### PR TITLE
 Added an action and reducer to accord for deleting a markdown

### PR DESCRIPTION
**Pull request by Zilula**
Added an action and reducer case that accounts for a deletion of a markdown file. Added a button to fire a function that calls said action and update the current store/state

### CODE REVIEW

File: src/components/SavedMarkdownList.js [hunk 1]
```diff
@@ -1,8 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-export function MarkdownTitles({ titles }) {
+
```
Comment (acrankin) [inline src/components/SavedMarkdownList.js]:
```suggestion
```

File: src/components/SavedMarkdownList.js [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
+
+export function MarkdownTitles({ titles, deleteMarkdown }) {
+
```
Comment (acrankin) [inline src/components/SavedMarkdownList.js]:
```suggestion
```

File: src/selectors/markdownListSelectors.js [hunk 3]
```diff
@@ -7,3 +7,7 @@ export const getTitles = state => {
     return markdown.title; 
   });
 };
+
```
Comment (acrankin) [inline src/selectors/markdownListSelectors.js:10]: No commented code!

File: src/reducers/markdownListReducer.js [hunk 4]
```diff
@@ -1,5 +1,4 @@
-import { CREATE_MARKDOWN } from '../actions/markdownListActions';
-
+import { CREATE_MARKDOWN, DELETE_MARKDOWN } from '../actions/markdownListActions';
```
Comment (MariahAdams) [inline src/reducers/markdownListReducer.js:1]: I would add a space after this import to keep syntax consistent across files.

**Review by MariahAdams — APPROVED** [review]
You're good to go once you clean up the comments on the PR!

**Review by caripizza — APPROVED** [review]
After confusing merge conflicts we are going to add the updates per Reviewers' comments AFTER we merge this branch in (those will come in a new branch/pr).

**Pull request merged**